src/main/java/pages/MovieDataPage.java

```diff
@@ -0,0 +1,33 @@
+package pages;
+
+import io.appium.java_client.MobileElement;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+import io.appium.java_client.pagefactory.iOSXCUITFindBy;
+
+public class MovieDataPage extends BasePage {
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(xpath = "//android.view.ViewGroup[1]/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup[3]/android.widget.ScrollView/android.view.ViewGroup/android.widget.TextView[1]")
```

Comment:
I will add several ids to the app to avoid using this horrible xpaths

Reply:
It would be a great idea since the application needs several ids